docs/source/pages/overview.rst

```diff
@@ -348,3 +348,24 @@ In practise this means that:
     val = metric.compute() # this value cannot be back-propagated
 
 A functional metric is differentiable if its corresponding modular metric is differentiable.
+
+.. _Metric kwargs:
+
+****************************
+Advance distributed settings
+****************************
+
+If you are running in a distributed environment, TorchMetrics will automatically take care of the distributed
+synchronization for you. However, the following three keyword arguments can be given to any modular metric for
+further control over the distributed synchronization:
+
+- ``dist_sync_on_step``: This argument is ``bool`` that indicates if the metric should syncronize between
+  different devices every time ``forward`` is called. Setting this to ``True`` is in general not recommended
+  as syncronization is an expensive operation to do after each batch.
+
+- ``process_group``: By default we syncronize across the *world* i.e. all proceses being computed on. You
+  can provide an ``torch._C._distributed_c10d.ProcessGroup`` in this argument to specify exactly what
```

Comment:
Is this really the only way to specify this? Also this doesn't do cross-links and if we don't do cross-links we probably don't need the whole path including the private C part of torch, right?

Reply:
I am really not sure, but from the docs it seems so:
https://pytorch.org/docs/stable/distributed.html#torch.distributed.all_gather